index.js

```diff
@@ -0,0 +1,6 @@
+let nombre = prompt("hola como te llamas")
+
+for (let i=0; i <5; i++)
+{
+    alert("hola " + nombre)
```

Comment:
esto se puede hacer con interpolación 
```suggestion
    alert(´hola ${nombre}´);
```